custom_components/waste_collection_schedule/waste_collection_schedule/source/olo_sk.py

```diff
@@ -0,0 +1,166 @@
+import datetime
+from waste_collection_schedule import Collection
+import requests
+
+TITLE = "OLO"  # Title will show up in README.md and info.md
+DESCRIPTION = "Source script for olo.sk"  # Describe your source
+# Insert url to service homepage. URL will show up in README.md and info.md
+URL = "https://www.olo.sk"
+TEST_CASES = {  # Insert arguments for test cases to be used by test_sources.py script
+    "Jantarova 47": {"street": 'Jantarova 47', "registrationNumber": '2441788'},
+    "Jasovska 8": {"street": 'Jasovska 8', "registrationNumber": '1353013'},
+    "Rovniankova 5": {"street": 'Rovniankova 5', "registrationNumber": ''},
+}
+
+API_URL = "https://olo-strapi-meilisearch.bratislava.sk/indexes/waste-collection-day/search"
+ICON_MAP = {   # Optional: Dict of waste types and suitable mdi icons
+    "DOMESTIC": "mdi:trash-can",
+    "RECYCLE": "mdi:recycle",
+    "KITCHEN": "mdi:coutertop",
+}
+
+WEKDAYS = {
+    "pondelok": "Monday",
+    "utorok": "Tuesday",
+    "streda": "Wednesday",
+    "štvrtok": "Thursday",
+    "piatok": "Friday",
+    "sobota": "Saturday",
+    "nedeľa": "Sunday"
+}
+
+#### Arguments affecting the configuration GUI ####
+
+HOW_TO_GET_ARGUMENTS_DESCRIPTION = {  # Optional dictionary to describe how to get the arguments, will be shown in the GUI configuration form above the input fields, does not need to be translated in all languages
+    "en": "HOW TO GET ARGUMENTS DESCRIPTION"
+}
```

Comment:
Either fill in correct information or remove this Variable. This placeholder text would be shown in the configuration GUI at the moment